src/main/scala/decider/Z3ProverStdIO.scala

```diff
@@ -235,10 +236,19 @@ class Z3ProverStdIO(uniqueId: String,
   }
 
   private def getModel(): Unit = {
-    if (Verifier.config.ideModeAdvanced()) {
+    if (Verifier.config.ideModeAdvanced() || Verifier.config.counterexample.toOption.isDefined) {
       writeLine("(get-model)")
-      val model = readModel().trim()
-      println(model + "\r\n")
+
+      if (Verifier.config.ideModeAdvanced()){

```

Comment:
@marcoeilers Any idea if the if-branch is still needed? A `println` of a model seems not really helpful, in particular in the context of something apparently related to the IDE (I don't know/remember who contributed that code).

Reply:
I don't know of anything that would use it. 
@aterga @ArquintL Do any of your student projects or your MSc thesis use the counterexample printed to the command line?

Reply:
My MSc thesis does not use anything printed to standard out. I've just double-checked the PR of my MSc thesis and original if branch was already there back then

Reply:
I'll just remove it then and we'll see if anyone complains.